nni/nas/strategy/hpo.py

```diff
@@ -56,45 +115,8 @@ class TPE(BaseStrategy):
     """
 
     def __init__(self):
```

Comment:
Add `seed` parameter?

Reply:
Why? How to set seed for TpeTuner?

Reply:
https://nni.readthedocs.io/en/stable/reference/hpo.html#nni.algorithms.hpo.tpe_tuner.TpeTuner
`TpeTuner(optimize_mode='minimize', seed=None, tpe_args=None)`